src/styles/public/print.scss

```diff
@@ -223,13 +233,58 @@
 			}
 		}
 
-		figure.print {
-			display: block !important;
+		figure {
+			display: none !important;
 
-			&.wp-block-media-text {
+			&.print,
+			&.wp-block-audio,
+			&.wp-block-gallery,
+			&.wp-block-image,
+			&.wp-block-pullquote,
+			&.wp-block-table {
+				display: block !important;
+				margin: 1rem 0 !important;
+				max-width: fit-content;
 
-				&__media {
-					display: block !important;
+				&.wp-block-media-text {
+
+					&__media {
+						display: block !important;
+					}
+				}
+
+				thead {
+					border-bottom: 2px solid #333 !important;
```

Comment:
probably should be using a sass variable for colors

Reply:
I've switched it to the sass $black variable which is not as soft as the previous setting but is also not fully black.